ui/apos/components/AposImportModal.vue

```diff
@@ -177,13 +189,23 @@ export default {
         return;
       }
       apos.bus.$emit('import-export-import-started');
+
+      const notifId = await apos.notify('apostrophe:uploading', {
+        type: 'progress',
+        icon: 'cloud-upload-icon',
+        interpolate: {
+          name: this.selectedFile.name
+        }
+      });
+
       bigUpload(`${this.universalModuleAction}/${this.action}`, {
         files: {
           file: this.selectedFile
         },
         body: {
           importDraftsOnly: this.checked.includes('importDraftsOnly')
-        }
+        },
+        progress: this.instantiateProcess(notifId)
       }).catch(() => {
```

Comment:
Use try/catch and await, don't mix in promise chain syntax where it can be avoided

Reply:
This is intentional and isn't related to this code. We don't use async / await because we don't want to block the process.
If you look at the code we want to close the modal before the process ends:

```javascript
      }).catch(() => {
        apos.bus.$emit('import-export-import-ended');
      });

      this.modal.showModal = false;
```

Reply:
Ok I guess I can do that, it's not code related to my PR but why not